specification/app/resource-manager/Microsoft.App/preview/2022-01-01-preview/EasyAuthConfigs.json

```diff
@@ -0,0 +1,706 @@
+        }
+      }
+    },
+    "LoginRoutes": {
+      "description": "The routes that specify the endpoints used for login and logout requests.",
+      "type": "object",
+      "properties": {
+        "logoutEndpoint": {
+          "description": "The endpoint at which a logout request should be made.",
+          "type": "string"
+        }
+      }
+    },
+    "HttpSettings": {
+      "description": "The configuration settings of the HTTP requests for authentication and authorization requests made against ContainerApp Authentication/Authorization.",
+      "type": "object",
+      "properties": {
+        "requireHttps": {
+          "description": "<code>false</code> if the authentication/authorization responses not having the HTTPS scheme are permissible; otherwise, <code>true</code>.",
+          "type": "boolean"
+        },
+        "routes": {
+          "$ref": "#/definitions/HttpSettingsRoutes",
+          "description": "The configuration settings of the paths HTTP requests."
+        }
+      }
+    },
+    "HttpSettingsRoutes": {
+      "description": "The configuration settings of the paths HTTP requests.",
+      "type": "object",
+      "properties": {
+        "apiPrefix": {
+          "description": "The prefix that should precede all the authentication/authorization paths.",
+          "type": "string"
+        }
+      }
+    },
+    "IdentityProviders": {
+      "description": "The configuration settings of each of the identity providers used to configure ContainerApp Authentication/Authorization.",
+      "type": "object",
+      "properties": {
+        "azureActiveDirectory": {
+          "$ref": "#/definitions/AzureActiveDirectory",
+          "description": "The configuration settings of the Azure Active directory provider."
+        },
+        "facebook": {
+          "$ref": "#/definitions/Facebook",
+          "description": "The configuration settings of the Facebook provider."
+        },
+        "gitHub": {
+          "$ref": "#/definitions/GitHub",
+          "description": "The configuration settings of the GitHub provider."
+        },
+        "google": {
+          "$ref": "#/definitions/Google",
+          "description": "The configuration settings of the Google provider."
+        },
+        "legacyMicrosoftAccount": {
+          "$ref": "#/definitions/LegacyMicrosoftAccount",
+          "description": "The configuration settings of the legacy Microsoft Account provider."
+        },
+        "twitter": {
+          "$ref": "#/definitions/Twitter",
+          "description": "The configuration settings of the Twitter provider."
+        },
+        "apple": {
+          "$ref": "#/definitions/Apple",
+          "description": "The configuration settings of the Apple provider."
+        }
+      }
+    },
+    "AzureActiveDirectory": {
+      "description": "The configuration settings of the Azure Active directory provider.",
+      "type": "object",
+      "properties": {
+        "enabled": {
```

Comment:
>enabled [](http://example.com/codeflow?start=9&length=7)

Consider a string enum instead of boolean. Similar comment for the other identity provider objects.